hooks/bootstrap.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2018 Shotgun Software Inc.
```

Comment:
This is the base hook implementation. This is what will be used by core to cache bundles during bootstrap.